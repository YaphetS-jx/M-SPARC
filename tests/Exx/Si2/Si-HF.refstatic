***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
     10.0000000000      12.0000000000      12.0000000000
     14.0000000000      12.0000000000      12.0000000000
Atomic forces (Ha/Bohr):
      0.0253372278       0.0000000013       0.0000000004
     -0.0253372278      -0.0000000013      -0.0000000004
