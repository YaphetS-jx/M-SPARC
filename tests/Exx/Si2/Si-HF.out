***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-Jan-2021  Start time: 19:39:50                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 24.000000 24.000000 24.000000 
FD_GRID: 80 80 80
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000950043
CHEB_DEGREE: 25
NSTATES: 6
EXCHANGE_CORRELATION: HF
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 0
MAXIT_FOCK: 100
TOL_FOCK: 1.00E-06
HYB_MIXING: 1.00
EXX_METHOD: FOURIER_SPACE
OUTPUT_FILE: ../tests/Exx/Si2/Si-HF.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.300000 (Bohr)
Output printed to                  :  ../tests/Exx/Si2/Si-HF.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.20 7.20 7.20
Number of atoms of type 1          :  2
Estimated total memory usage       :  2.01 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.7855095801E+00        1.597E-01        20.100
2            -3.8018897874E+00        9.932E-02        18.796
3            -3.8198415087E+00        6.061E-02        18.733
4            -3.8240974405E+00        4.936E-02        19.387
5            -3.8248317156E+00        3.945E-02        18.550
6            -3.8271720115E+00        1.881E-02        18.356
7            -3.8266531729E+00        1.924E-02        17.630
8            -3.8277815451E+00        9.895E-03        18.682
9            -3.8281886138E+00        4.185E-03        17.732
10           -3.8281630297E+00        3.786E-03        16.594
11           -3.8282404267E+00        1.112E-03        15.693
12           -3.8282404139E+00        6.700E-04        15.579
13           -3.8282425023E+00        2.938E-04        14.511
14           -3.8282429055E+00        1.361E-04        14.634
15           -3.8282429412E+00        7.210E-05        12.391
16           -3.8282429344E+00        5.138E-05        12.832
17           -3.8282428899E+00        3.519E-05        11.297
18           -3.8282429487E+00        1.939E-05        12.470
19           -3.8282428886E+00        1.102E-05        12.300
20           -3.8282429579E+00        5.253E-06        11.137
21           -3.8282429219E+00        1.779E-06        10.681
22           -3.8282429228E+00        1.065E-06        9.376
23           -3.8282429324E+00        6.917E-07        8.034
Total number of SCF: 23    
No.1 Exx outer loop:
1            -4.4640394250E+00        7.688E-02        165.318
2            -4.4738757919E+00        2.500E-02        54.377
3            -4.4738141259E+00        1.962E-02        55.672
4            -4.4512141644E+00        9.198E-02        57.406
5            -4.4706073068E+00        3.967E-02        57.983
6            -4.4582391279E+00        7.616E-02        57.231
7            -4.4743335013E+00        1.434E-02        55.859
8            -4.4748459118E+00        5.060E-03        55.994
9            -4.4741620037E+00        1.631E-02        55.379
10           -4.4749012198E+00        2.624E-03        55.848
11           -4.4749189352E+00        8.097E-04        56.523
12           -4.4749150027E+00        1.420E-03        53.455
13           -4.4749204651E+00        3.486E-04        52.969
14           -4.4749208013E+00        5.802E-05        52.439
15           -4.4749207507E+00        2.180E-05        52.520
16           -4.4749208035E+00        1.616E-05        51.284
17           -4.4749207778E+00        9.485E-06        49.941
18           -4.4749208012E+00        4.100E-06        48.872
19           -4.4749207968E+00        2.179E-06        48.329
20           -4.4749207930E+00        2.189E-06        47.588
21           -4.4749207891E+00        9.177E-07        47.705
Total number of SCF: 21    
 Eex = 1.69749218
 Etot = -7.25234940
No.2 Exx outer loop:
1            -4.5018901658E+00        4.954E-02        160.360
2            -4.5034101331E+00        2.786E-02        53.975
3            -4.5034041514E+00        2.788E-02        52.702
4            -4.5039836474E+00        1.101E-02        52.540
5            -4.5040028518E+00        9.601E-03        53.897
6            -4.5040636882E+00        5.231E-03        51.660
7            -4.5040766912E+00        3.581E-03        52.114
8            -4.5040884580E+00        2.978E-04        51.567
9            -4.5040886174E+00        8.431E-05        51.957
10           -4.5040886129E+00        2.819E-05        51.092
11           -4.5040885939E+00        7.237E-06        49.612
12           -4.5040886102E+00        2.498E-06        47.682
13           -4.5040886214E+00        9.257E-07        46.220
Total number of SCF: 13    
 Eex = 1.69693168
 Etot = -7.31124557
No.3 Exx outer loop:
1            -4.5792354245E+00        7.995E-03        159.805
2            -4.5792773644E+00        4.615E-03        52.334
3            -4.5793014834E+00        2.352E-04        50.951
4            -4.5793015836E+00        8.638E-05        52.060
5            -4.5793015870E+00        3.694E-05        49.996
6            -4.5793016222E+00        1.474E-05        49.046
7            -4.5793016105E+00        5.818E-06        48.876
8            -4.5793016120E+00        1.745E-06        47.162
9            -4.5793016067E+00        5.555E-07        46.319
Total number of SCF: 9     
 Eex = 1.80269097
 Etot = -7.35591225
No.4 Exx outer loop:
1            -4.5852295893E+00        3.206E-03        162.629
2            -4.5852373163E+00        1.989E-03        53.298
3            -4.5852421933E+00        3.305E-05        49.570
4            -4.5852422318E+00        2.359E-05        50.421
5            -4.5852422151E+00        1.036E-05        50.046
6            -4.5852422474E+00        6.877E-06        49.250
7            -4.5852422526E+00        1.900E-06        46.204
8            -4.5852422707E+00        8.211E-07        47.219
Total number of SCF: 8     
 Eex = 1.81073524
 Etot = -7.35974930
No.5 Exx outer loop:
1            -4.5878775270E+00        1.369E-03        161.670
2            -4.5878794024E+00        8.846E-04        51.510
3            -4.5878804635E+00        4.739E-05        50.059
4            -4.5878804846E+00        1.581E-05        50.104
5            -4.5878805049E+00        8.302E-06        48.965
6            -4.5878805074E+00        3.093E-06        47.403
7            -4.5878804894E+00        2.056E-06        48.058
8            -4.5878804661E+00        7.408E-07        45.635
Total number of SCF: 8     
 Eex = 1.81446805
 Etot = -7.36129288
No.6 Exx outer loop:
1            -4.5890899351E+00        6.257E-04        159.983
2            -4.5890903920E+00        4.050E-04        50.951
3            -4.5890906724E+00        1.092E-05        50.304
4            -4.5890906926E+00        2.250E-06        49.838
5            -4.5890907187E+00        3.436E-06        48.824
6            -4.5890907150E+00        9.600E-07        45.790
Total number of SCF: 6     
 Eex = 1.81621610
 Etot = -7.36196533
No.7 Exx outer loop:
1            -4.5896540569E+00        3.014E-04        159.917
2            -4.5896541810E+00        1.899E-04        49.927
3            -4.5896542981E+00        1.026E-05        48.913
4            -4.5896542891E+00        3.711E-06        48.530
5            -4.5896543125E+00        1.502E-06        47.613
6            -4.5896542971E+00        5.685E-07        45.947
Total number of SCF: 6     
 Eex = 1.81703867
 Etot = -7.36226992
No.8 Exx outer loop:
1            -4.5899190552E+00        1.498E-04        158.185
2            -4.5899190804E+00        9.054E-05        52.187
3            -4.5899191438E+00        3.726E-06        49.305
4            -4.5899191302E+00        7.780E-07        48.074
Total number of SCF: 4     
 Eex = 1.81742761
 Etot = -7.36241065
No.9 Exx outer loop:
1            -4.5900445369E+00        7.528E-05        158.871
2            -4.5900445142E+00        4.341E-05        49.196
3            -4.5900445699E+00        1.418E-06        48.780
4            -4.5900445959E+00        5.425E-07        47.096
Total number of SCF: 4     
 Eex = 1.81761215
 Etot = -7.36247704
No.10 Exx outer loop:
1            -4.5901039632E+00        3.793E-05        158.386
2            -4.5901039327E+00        2.086E-05        49.370
3            -4.5901039603E+00        6.275E-07        47.667
Total number of SCF: 3     
 Eex = 1.81769993
 Etot = -7.36250799
No.11 Exx outer loop:
1            -4.5901323943E+00        1.915E-05        158.096
2            -4.5901323814E+00        1.022E-05        48.526
3            -4.5901323923E+00        3.601E-07        47.140
Total number of SCF: 3     
 Eex = 1.81774193
 Etot = -7.36252286
No.12 Exx outer loop:
1            -4.5901459622E+00        9.571E-06        157.408
2            -4.5901459498E+00        4.959E-06        47.303
3            -4.5901459607E+00        3.268E-07        46.822
Total number of SCF: 3     
 Eex = 1.81776197
 Etot = -7.36252995
No.13 Exx outer loop:
1            -4.5901524365E+00        4.774E-06        158.251
2            -4.5901524168E+00        2.456E-06        47.082
3            -4.5901524348E+00        2.661E-07        45.623
Total number of SCF: 3     
 Eex = 1.81777154
 Etot = -7.36253333
No.14 Exx outer loop:
1            -4.5901555262E+00        2.419E-06        158.898
2            -4.5901554882E+00        1.255E-06        46.742
3            -4.5901555144E+00        1.848E-07        45.547
Total number of SCF: 3     
 Eex = 1.81777616
 Etot = -7.36253487
No.15 Exx outer loop:
1            -4.5901570347E+00        1.391E-06        157.973
2            -4.5901570308E+00        7.837E-07        46.146
3            -4.5901570473E+00        1.493E-07        45.326
Total number of SCF: 3     
 Eex = 1.81777844
 Etot = -7.36253566
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.6812678285E+00 (Ha/atom)
Total free energy                  : -7.3625356570E+00 (Ha)
Band structure energy              : -3.4261919845E+00 (Ha)
Exchange correlation energy        : -0.0000000000E+00 (Ha)
Self and correction energy         : -4.8308993097E+01 (Ha)
Entropy*kb*T                       : -1.7990336522E-10 (Ha)
Fermi level                        : -9.3946890438E-02 (Ha)
Average force                      :  2.5337227794E-02 (Ha/Bohr)
Maximum force                      :  2.5337227794E-02 (Ha/Bohr)
Time for force calculation         :  2.975 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  7615.098 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
